NEW_MODULE_TECH_EVAL.MD

```diff
@@ -71,3 +71,22 @@ Information gathered includes:
       * If a meeting is required or desired, it's the responsibility of the contributor(s) to find a time that works and set the meeting up with the review team.  The regular Technical Council meeting ***will not*** be used for this purpose.
 1. If applicable, the contributor(s) resolve all issues and request a re-review.  
     * * JIRA workflow: the ticket is transitioned from  **REJECTED** to **SUBMITTED**
+
+### Roles
```

Comment:
Given that roles are being defined, I think it would make sense for the process definition (above) to be written using these roles.

At the moment, it is either written in a passive form referring to the Councils and development team or it uses roles that aren't in this list e.g. contributor, point of contact, release coordinator.

Reply:
Good point.